azext_iot/operations/_mqtt.py

```diff
@@ -30,12 +31,19 @@ def __init__(
         self.default_data_encoding = 'utf-8'
         self.init_reported_properties = init_reported_properties
 
-    def send_d2c_message(self, message_text, properties=None):
+    def send_d2c_message(
+        self, message_text, properties=None
+    ):
+        ensure_azure_namespace_path()
```

Comment:
`ensure_azure_namespace_path()` needs to be called just once prior to `azure.*` imports

Reply:
Updated